webapp/src/css/sidebar-filter.less

```diff
@@ -99,6 +99,68 @@
       overflow-y: auto;
       overflow-x: hidden;
 
+      .filter-options-container {
```

Comment:
![Screenshot 2024-07-16 at 12 44 07 PM](https://github.com/user-attachments/assets/a512d480-e182-4978-8fd4-fba8cbdf5c1e)
Long facility names is making the radio button an oval, it should keep as circle

Reply:
Also in desktop
<img width="748" alt="Screenshot 2024-07-16 at 12 46 40 PM" src="https://github.com/user-attachments/assets/87fb5066-3dc2-4681-9e10-49a50a33e21a">

Reply:
This is why I was using the `!important` what's another way I can ensure it is always a circle without using important ?

Reply:
Make the selector more specific

Reply:
this fixes it: `min-width: 20px;` in the input